### PR TITLE
Update check-in owners steps on objective-delegation.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/Viva/issues/536

### CODE REVIEW

File: Viva/goals/objective-delegation.md [hunk 1]
```diff
@@ -54,11 +54,11 @@ If you want another user to check in and update the progress of an OKR, you can
 
 ### How to assign responsibility for check-ins
 
-1. Log in to Viva Goals and select the **+** button on the top panel to create a new OKR.
+1. Log in to Viva Goals and select the **Add Objective** button to create a new OKR. If you want to modify an existing one, you can select it, click **More** and then **Edit**.
 
-2. Identify the objective/key result you want to share with another individual. Then select the new owners via the **Owner** option. You can assign multiple owners for the OKR.
+2. Identify the objective/key result you want to share with another individual. Then select the new owners via the **Owner** option under **Details** section. You can assign multiple owners for the OKR.
```
Comment (JohanFreelancer9) [inline Viva/goals/objective-delegation.md:59]:
```suggestion
2. Identify the objective/key result you want to share with another individual, then select the new owners via the **Owner** option in the **Details** section. You can assign multiple owners for the OKR.
```

File: Viva/goals/objective-delegation.md [hunk 2]
```diff
@@ -54,11 +54,11 @@ If you want another user to check in and update the progress of an OKR, you can
...
 
-3. After you assign owners, use the **Who is responsible for making check-ins?** dropdown to select the user who will be responsible for making check-ins. That user will start receiving check-in reminders.
+3. After you assign owners, use the **Check-in owner** dropdown under **Progress** section to select the user who will be responsible for making check-ins. That user will start receiving check-in reminders.
```
Comment (JohanFreelancer9) [inline Viva/goals/objective-delegation.md:61]:
```suggestion
3. After you assign owners, use the **Check-in owner** dropdown in the **Progress** section to select the user who will be responsible for making check-ins. That user will start receiving check-in reminders.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dariomws

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@RaSanders-MSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by RaSanders-MSFT** [comment]
Changes look good!

Do I need to add the sign off label, or press the "merge pull request" button?

**Comment by dariomws** [comment]
@RaSanders-MSFT, merge pull request, please. Thanks!

**Pull request merged**